BUILDING.md

```diff
@@ -66,6 +66,12 @@ note1 - The gcc4.8-libs package needs to be installed, because node
   In "Git bash" if you call the node shell alias (`node` without the `.exe`
   extension), `winpty` is used automatically.
 
+*Note*: The Windows Subsystem for Linux (WSL) is not directly supported, but
+  the GNU/Linux build process and binaries should work. The community will
+  only address issues that reproduce on native GNU/Linux systems, issues that
+  only reproduce on WSL should be reported in the
+  [WSL issue tracker](https://github.com/Microsoft/WSL/issues).
```

Comment:
It would be good to mention executing `node.exe` in WSL is not supported either.

Reply:
```